labman/db/process.py

```diff
@@ -3545,6 +3558,13 @@ def generate_prep_information(self):
                     'EXPERIMENT_DESIGN_DESCRIPTION'
             }
             df.rename(index=str, columns=mv, inplace=True)
+            # as orig_name2 has been transformed into Orig_name, and
+            # the original orig_name column has been used to generate
+            # sample name, there is no longer a purpose for the
+            # original orig_name column, hence drop it from the final
```

Comment:
For (verbose) clarity, could you change "orig_name column has been used to generate sample name" to "orig_name column has been used to generate df.index, which will be used as the sample name"?

Reply:
sure thing!